【arkctl】文件找不到时不需要报错出堆栈

当找不到文件时，可以不用抛出异常堆栈，当前会抛出如下堆栈

![image](https://github.com/koupleless/koupleless/assets/3754074/c20c8f10-e384-47f4-b4f1-29b6a922e635)

```shell
panic: open /Users/liuquanwang/root/code/intl/listing-limit-server/listing-limit-api/target/listing-limit-api-2.0.0-SNAPSHOT-ark-biz.jar: no such file or directory

goroutine 1 [running]:
github.com/koupleless/arkctl/common/runtime.MustReturnResult[...](...)
        /github/workspace/common/runtime/runtime_util.go:25
github.com/koupleless/arkctl/v1/service/ark.parseJarBizModel({0x102998530, 0x14000114570}, {0x140001de120, 0x83})
        /github/workspace/v1/service/ark/biz_util.go:40 +0x59c
github.com/koupleless/arkctl/v1/service/ark.ParseBizModel({0x102998530, 0x14000114570}, {0x140001de120, 0x83})
        /github/workspace/v1/service/ark/biz_util.go:81 +0x9c
github.com/koupleless/arkctl/v1/cmd/deploy.execParseBizModel(0x14000114570)
        /github/workspace/v1/cmd/deploy/deploy.go:171 +0x220
github.com/koupleless/arkctl/v1/cmd/deploy.executeDeploy(0x140001d0200?, {0x14000120b90?, 0x4?, 0x1027cd687?})
        /github/workspace/v1/cmd/deploy/deploy.go:429 +0x88
github.com/spf13/cobra.(*Command).execute(0x102cf5b60, {0x14000120b70, 0x1, 0x1})
        /go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:860 +0x53c
github.com/spf13/cobra.(*Command).ExecuteC(0x102cf58e0)
        /go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:974 +0x318
github.com/spf13/cobra.(*Command).Execute(...)
        /go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:902
github.com/koupleless/arkctl/v1/cmd/root.Execute()
        /github/workspace/v1/cmd/root/root.go:41 +0x24
main.main()
        /github/workspace/main.go:28 +0x28
```

--- a/common/style/style.go
+++ b/common/style/style.go
@@ -25,3 +25,13 @@ func InfoPrefix(prefix string) *pterm.PrefixPrinter {
 		MessageStyle: pterm.Info.MessageStyle,
 	}
 }
+
+func ErrorPrefix(prefix string) *pterm.PrefixPrinter {
+	return &pterm.PrefixPrinter{
+		Prefix: pterm.Prefix{
+			Text:  prefix,
+			Style: pterm.Error.Prefix.Style,
+		},
+		MessageStyle: pterm.Error.MessageStyle,
+	}
+}

--- a/v1/cmd/deploy/deploy.go
+++ b/v1/cmd/deploy/deploy.go
@@ -20,6 +20,7 @@ package deploy
 import (
 	"context"
 	"encoding/json"
+	"errors"
 	"fmt"
 	"github.com/koupleless/arkctl/common/osutil"
 	"os"
@@ -169,6 +170,10 @@ func execParseBizModel(ctx *contextutil.Context) bool {
 	}
 
 	bizModel, err := ark.ParseBizModel(ctx, fileutil.FileUrl(bundlePath))
+	if errors.Is(err, os.ErrNotExist) {
+		style.ErrorPrefix("File Not Exist").Println(defaultArg)
+		return false
+	}
 	if err != nil {
 		pterm.Error.PrintOnError(fmt.Errorf("failed to parse bundle: %s", err))
 		return false

--- a/v1/service/ark/biz_util.go
+++ b/v1/service/ark/biz_util.go
@@ -17,9 +17,11 @@ package ark
 import (
 	"archive/zip"
 	"context"
+	"errors"
 	"github.com/koupleless/arkctl/common/osutil"
 	"github.com/koupleless/arkctl/common/runtime"
 	"io"
+	"os"
 	"strings"
 
 	"github.com/koupleless/arkctl/common/fileutil"
@@ -36,8 +38,11 @@ func parseJarBizModel(ctx context.Context, bizUrl fileutil.FileUrl) (model *BizM
 	defer runtime.RecoverFromError(&err)
 	fileUtil := fileutil.DefaultFileUtil()
 	localPath := runtime.MustReturnResult(fileUtil.Download(ctx, bizUrl))
-
-	zipReader := runtime.MustReturnResult(zip.OpenReader(localPath[len(osutil.GetLocalFileProtocol()):]))
+	readerJar, err := zip.OpenReader(localPath[len(osutil.GetLocalFileProtocol()):])
+	if errors.Is(err, os.ErrNotExist) {
+		return nil, err
+	}
+	zipReader := runtime.MustReturnResult(readerJar, err)
 	defer zipReader.Close()
 
 	bizName := ""

--- a/v1/service/ark/biz_util_test.go
+++ b/v1/service/ark/biz_util_test.go
@@ -19,6 +19,7 @@ import (
 	"context"
 	"fmt"
 	"github.com/koupleless/arkctl/common/osutil"
+	assert2 "github.com/stretchr/testify/assert"
 	"io"
 	"os"
 	"path/filepath"
@@ -30,6 +31,14 @@ import (
 	"github.com/google/uuid"
 	"github.com/magiconair/properties/assert"
 )
+
+func TestParseBizModel_FileNotExist(t *testing.T) {
+	_, err := ParseBizModel(
+		context.Background(),
+		"file://ll.jar",
+	)
+	assert2.ErrorIs(t, err, os.ErrNotExist)
+}
 
 func TestParseBizModel_LocalJar(t *testing.T) {
 	// creating a mock jar file

